infra/terraform/external_function/snowalert/lambda-code/protocols/https.py

```diff
@@ -68,8 +66,6 @@ def https(
             req_headers['Authorization'] = make_basic_header(req_auth['basic'])
         elif 'bearer' in req_auth:
             req_headers['Authorization'] = f"Bearer {req_auth['bearer']}"
-        elif 'authorization' in req_auth:
```

Comment:
same here -- looks like you're trying to undo some features I added, please keep them